website/docs/guides/troubleshooting.mdx

```diff
@@ -203,9 +203,61 @@ you can make the webview background transparent using the following config:
 	})
 ```
 
+## I get a "Microsoft Edge can't read or write to its data directory" error when running my program as admin on Windows
+
+You set your program to require admin permissions and it worked great! Unfortunately, some users are seeing a "Microsoft Edge can't read or write to its data directory" error when running it.
+
+When a Windows machine has two local accounts:
+
+- Alice, an admin
+- Bob, a regular user
+
+Bob sees a UAC prompt when running your program. Bob enters Alice's admin credentials into this prompt. The app launches with admin permissions under Alice's account.
+
+Wails instructs WebView2 to store user data at the specified `WebviewUserDataPath`. It defaults to `%APPDATA%\[BinaryName.exe]`. 
+
+Because the application is running under Alice's account, `%APPDATA%\[BinaryName.exe]` resolves to `C:\Users\Alice\AppData\Roaming\[BinaryName.exe]`.
+
+WebView2 [creates some child processes under Bob's logged-in account instead of Alice's admin account](https://github.com/MicrosoftEdge/WebView2Feedback/issues/932#issue-807464179). Since Bob cannot access `C:\Users\Alice\AppData\Roaming\[BinaryName.exe]`, the "Microsoft Edge can't read or write to its data directory" error is shown.
+
+Possible solution #1:
+
+Refactor your application to work without constant admin permissions. If you just need to perform a small set of admin tasks (such as running an updater), you can run your application with the minimum permissions and then use the `runas` command to run these tasks with admin permissions as needed:
+
+```go
+//go:build windows
+
+package sample
+
+import (
+    "golang.org/x/sys/windows"
+    "syscall"
+)
+
+// Calling RunAs("C:\path\to\my\updater.exe") shows Bob a UAC prompt. Bob enters Alice's admin credentials. The updater launches with admin permissions under Alice's account.
+func RunAs(path string) error {
+    verbPtr, _ := syscall.UTF16PtrFromString("runas")
+    exePtr, _ := syscall.UTF16PtrFromString(path)
+    cwdPtr, _ := syscall.UTF16PtrFromString("")
+    argPtr, _ := syscall.UTF16PtrFromString("")
+
+    var showCmd int32 = 1 //SW_NORMAL
+
+    err := windows.ShellExecute(0, verbPtr, exePtr, argPtr, cwdPtr, showCmd)
+    if err != nil {
+        return err
+    }
+    return nil
+}
+```
+
+Possible solution #2: change `WebviewUserDataPath` to a path writeable by both Alice and Bob. 
```

Comment:
Tried this but it didn't work. Error: https://user-images.githubusercontent.com/93655205/268789148-358c596b-dd25-4207-8d20-52b8c66ca281.png

Reply:
Interesting. I took time to setup a VM and can confirm that it also doesn't work for me, even if using a non-userprofile directory ACL'd to everyone.

However, if I use an elevated Git bash to launch the program (still using the ACL'd directory), all the child Webview2 processes launch under Bob's account and the application works :thinking: 

![image](https://github.com/wailsapp/wails/assets/852873/d5f74319-9c55-43c1-a02f-2fc8924191e2)

I'm using the default template with this main.go:

```go
package main

import (
	"embed"
	"os"

	"github.com/hectane/go-acl"
	"github.com/wailsapp/wails/v2"
	"github.com/wailsapp/wails/v2/pkg/options"
	"github.com/wailsapp/wails/v2/pkg/options/assetserver"
	"github.com/wailsapp/wails/v2/pkg/options/windows"
)

//go:embed all:frontend/dist
var assets embed.FS

func main() {
	var windowsOptions *windows.Options
	if len(os.Args) > 1 {
		dir := os.Args[1]
		println("Setting data dir to", dir)
		if err := os.MkdirAll(dir, os.ModePerm); err != nil {
			println("Failed creating dir:", err)
		}
		if err := acl.Chmod(dir, 0777); err != nil {
			println("Failed setting ACL on dir:", err)
		}
		windowsOptions = &windows.Options{
			WebviewUserDataPath: dir,
		}
	}

	// Create an instance of the app structure
	app := NewApp()

	// Create application with options
	err := wails.Run(&options.App{
		Title:  "sample-data-dir",
		Width:  1024,
		Height: 768,
		AssetServer: &assetserver.Options{
			Assets: assets,
		},
		BackgroundColour: &options.RGBA{R: 27, G: 38, B: 54, A: 1},
		OnStartup:        app.startup,
		Bind: []interface{}{
			app,
		},
		Windows: windowsOptions,
	})

	if err != nil {
		println("Error:", err.Error())
	}
}
```

Reply:
Hello,

As a disclaimer, let it be known that I have understanding of the Windows security model.

The tokens used when running the program with Git Bash seem to have more permissions, allowing it to work without receiving a `80010108 The object invoked has disconnected from its clients` error:

![image](https://github.com/wailsapp/wails/assets/852873/03ec5513-a684-4d0a-9d0a-0fe9a247983b)

Left is the binary running regularly. Right is the binary running from an elevated Git Bash.

I made a PoC Wails app that re-runs itself using a token with the same permissions. It works in the above Bob/Alice split-account scenario: https://github.com/AlbinoDrought/sample-split-account-admin-wails/commit/823a03de826c7e1948fbda7e44b30083f774c9a0

Does this work for you as well?

Cheers

Reply:
Added to troubleshooting doc in https://github.com/wailsapp/wails/compare/4c44b6d66b0029a69818d0b3f2bdb4a840f7279f..afc1750ef6aadde2713c7150d7d11a0b68ac008d